File: testprojects/scala_main/pipelines/SCALA_UNITY_CATALOG/code/src/main/scala/io/prophecy/pipelines/scala_unity_catalog/graph/Filter_1.scala
package io.prophecy.pipelines.scala_unity_catalog.graph

import io.prophecy.libs._
import io.prophecy.pipelines.scala_unity_catalog.config.ConfigStore._
import io.prophecy.pipelines.scala_unity_catalog.config.Context
import io.prophecy.pipelines.scala_unity_catalog.udfs.UDFs._
import io.prophecy.pipelines.scala_unity_catalog.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Filter_1 {
  def apply(context: Context, in: DataFrame): DataFrame = in.filter(lit(true))
}

File: testprojects/streaming/python/pipelines/ORCOnly/code/orconly/graph/Filter_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from orconly.config.ConfigStore import *
from orconly.udfs.UDFs import *

def Filter_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.filter(lit(True))
